rudder-core/src/main/resources/Migration/ldapMigration-3.1.x-3.1.14-3.2.7-disable-js-directive.ldif

```diff
@@ -1,39 +0,0 @@
-###############################################################################
```

Comment:
That migration should still be used !! (migratin from 3.1 to 4.0 should keep it disabled)